gensrc/thrift/FrontendService.thrift

```diff
@@ -847,7 +847,10 @@ struct TLoadTxnBeginRequest {
     // The real value of timeout should be i32. i64 ensures the compatibility of interface.
     10: optional i64 timeout
     11: optional Types.TUniqueId request_id
-    101: optional string warehouse   // begin from 101, in case of conflict with other's change
+    
+    // begin from 101, in case of conflict with other's change
+    101: optional string warehouse
```

Comment:
will this be deprecated as well?

Reply:
Yes. In the future, we hope to use the backend method to implicitly convey information about the warehouse. Currently, the code is compatible with both warehouse and backend, with backend having a higher priority.

Reply:
have done all suggestions.